### ISSUE TITLE
TerminatorRuntime doesn't work with empty archive

### ISSUE THREAD

**Issue by mb706** [mlr-org/bbotk#128]
Should probably be automatically tested with all terminators.

```r
> trm("run_time")$is_terminated(Archive$new(ps(x = p_dbl()), ps(x = p_dbl())))
Error in as.POSIXct.default(time2) : 
  do not know how to convert 'time2' to class “POSIXct”
```

**Comment by be-marc** [comment]
All terminators work with an empty archive. `TerminatorRunTime` needs `Archive$start_time` which is set by the `Optimizer`. We could add an error message if `Archive$start_time = NULL`.

**Comment by be-marc** [comment]
The unit test check now if all terminators work with an empty archive. `TerminatorRunTime` throws an error if `Archive$start_time` is not set and documentation mentions that `archive$start_time` must be set. Closed by 1c89ab8eebb8cc07e2bb070ed23527cf1240a152.
